packages/thirdweb/src/adapters/viem.test.ts

```diff
@@ -62,4 +63,16 @@ describe("walletClient.toViem", () => {
     expect(txHash).toBeDefined();
     expect(txHash.slice(0, 2)).toBe("0x");
   });
+
+  test("should get address on live chain", async () => {
```

Comment:
This may fail on fork test runs, let's see, if it does we should skip this test if TW_SECRET_KEY env var isn't set